pkg/gen/typegen.go

```diff
@@ -1037,6 +1038,18 @@ func createGroups(definitionsJSON map[string]interface{}, opts groupOpts) []*Gro
 						panic(fmt.Sprintf("Unsupported language '%s'", opts.language))
 					}
 
+					// Set Secret input fields to pulumi.secret
```

Comment:
Note to self: the easiest way to support this sort of codegen from `pulumi/pulumi/pkg/codegen/schema` is to allow inputs to be marked secret.